odl/util/phantom.py

```diff
@@ -526,6 +543,150 @@ def rect(x):
     return out.ufunc.minimum(1, out=out)
 
 
+def cuboid(discr_space, begin, end):
+    """Rectangular cuboid.
+
+    Parameters
+    ----------
+    discr_space : `Discretization`
+        Discretized space in which the phantom is supposed to be created
+    begin : array-like or `float` in [0, 1]
+        The lower left corner of the cuboid within the space grid relative
+        to the extend of the grid
+    end : array-like or `float` in [0, 1]
+        The upper right corner of the cuboid within the space grid relative
+        to the extend of the grid
+
+    Returns
+    -------
+    phantom : `LinearSpaceVector`
+        Returns an element in ``discr_space``
+
+    Example
+    -------
+    >>> import odl
+    >>> space = odl.uniform_discr(0, 1, 6, dtype='float32')
+    >>> print(cuboid(space, 0.5, 1))
+    [0.0, 0.0, 0.0, 1.0, 1.0, 1.0]
+    >>> space = odl.uniform_discr([0, 0], [1, 1], [4, 6], dtype='float32')
+    >>> print(cuboid(space, [0.25, 0], [0.75, 0.5]))
+    [
+        [0.0, 0.0, 0.0, 0.0, 0.0, 0.0],
+        [1.0, 1.0, 1.0, 0.0, 0.0, 0.0],
+        [1.0, 1.0, 1.0, 0.0, 0.0, 0.0],
+        [0.0, 0.0, 0.0, 0.0, 0.0, 0.0]
+    ]
+    """
+    ndim = discr_space.ndim
+    shape = discr_space.shape
+
+    if np.isscalar(begin):
+        begin = (begin,) * ndim
+    if np.isscalar(end):
+        end = (end,) * ndim
+
+    # Create phantom
+    phan = np.zeros(shape)
+
+    slice1 = [slice(None)] * ndim
+
+    for nn in range(ndim):
+        start = np.floor(begin[nn] * shape[nn]).astype(int)
+        stop = np.ceil(end[nn] * shape[nn]).astype(int)
+
+        slice1[nn] = slice(start, stop)
+
+    phan[slice1] = 1
+
+    return discr_space.element(phan)
+
+
+def indicate_proj_axis(discr_space, scale_structures=0.5):
+    """Phantom indicating along which axis it is projected.
```

Comment:
Very useful guy, I was thinking of this as well, but perhaps we could make them abit more separated, this way it would be easier to use with conebeam or other geometries that are not "as good".